sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/MutationGroupEncoderTest.java

```diff
@@ -528,6 +528,32 @@ public void dateKeys() throws Exception {
     verifyEncodedOrdering(schema, "test", keys);
   }
 
+  @Test
+  // Test for Jira issue BEAM-4862
+  public void decodeTimestampMutationGroup() {
```

Comment:
Please, split this test into two:
decodeTimestampCanDecodeTimestampMinValue and decodeTimestampCanDecodeTimestampMaxValue.

These names will give detailed description on what this test does. Splitting will allow to easily understand failing setup by reading test name only.